packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -203,12 +203,19 @@ export class Database {
     } else {
       const tinaSchema = await this.getSchema(this.level)
       const extension = path.extname(filepath)
-      const contentObject = await this.level
-        .sublevel<string, Record<string, any>>(
-          CONTENT_ROOT_PREFIX,
-          SUBLEVEL_OPTIONS
-        )
-        .get(normalizePath(filepath))
+      let contentObject
+      try {
+        contentObject = await this.level
+          .sublevel<string, Record<string, any>>(
+            CONTENT_ROOT_PREFIX,
+            SUBLEVEL_OPTIONS
+          )
+          .get(normalizePath(filepath))
+      } catch (e: any) {
+        if (e.code !== 'LEVEL_NOT_FOUND') {
```

Comment:
Seems we have this try/catch a few times. Should we be swallowing this error at a different level to avoid needing to always wrap it in this try catch?

Reply:
I'll make a follow-up PR to wrap the level implementation and centralize this